dev/com.ibm.ws.org.apache.cxf.cxf.rt.transports.http.3.2/src/org/apache/cxf/transport/http/URLConnectionHTTPConduit.java

```diff
@@ -124,6 +126,20 @@ private HttpURLConnection createConnection(Message message, Address address, HTT
         if (clientParameters == null) {
             clientParameters = tlsClientParameters;
         }
+
+        // Liberty start    
+	boolean isRestMessage =
+            PropertyUtils.isTrue(message.getExchange().get(org.apache.cxf.message.Message.REST_MESSAGE));
```

Comment:
Can you confirm that `org.apache.cxf.rest.message` is present on MP Rest Client requests in addition to vanilla JAX-RS request please?